receiver/snowflakereceiver/factory.go

```diff
@@ -13,3 +13,64 @@
 // limitations under the License.
 
 package snowflakereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/snowflakereceiver"
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver"
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/snowflakereceiver/internal/metadata"
+)
+
+const (
+	typeStr         = "snowflakemetrics"
```

Comment:
This should be aligned with the module name which is `snowflake`

Reply:
heard